engines/python/setup/djl_python/input_parser.py

```diff
@@ -229,15 +229,17 @@ def parse_adapters(request_input: TextInput, input_item: Input,
 def _fetch_adapters_from_input(input_map: dict, input_item: Input):
     adapters_per_item = []
     if "adapters" in input_map:
-        adapters_per_item = input_map.pop("adapters", [])
+        adapters_per_item = (input_map.pop("adapters"), None)
 
     # check content, possible in workflow approach
     if input_item.contains_key("adapter"):
-        adapters_per_item = input_item.get_as_string("adapter")
+        adapters_per_item = (input_item.get_as_string("adapter"), None)
 
     # check properties, possible from header
-    if "adapter" in input_item.get_properties():
-        adapters_per_item = input_item.get_properties()["adapter"]
+    if "X-Amzn-SageMaker-Adapter-Identifier" in input_item.get_properties():
```

Comment:
do we need to parse these header here? I thought we already parse and convert on the frontend?